sdk/cognitivelanguage/azure-ai-language-conversations/samples/README.md

```diff
@@ -0,0 +1,91 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-ai-language-conversations
```

Comment:
The product slug indicates where the samples should be published. To me, azure-language-understanding seems correct. Full list here: https://review.docs.microsoft.com/en-us/help/contribute/metadata-taxonomies?branch=master#product

```suggestion
  - azure-language-understanding
```